diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2fecaf8..714618d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -411,9 +411,7 @@ endif()
 
 # Find Qt
 set(QT_COMPONENTS Core Network Concurrent Gui Svg Widgets Test LinguistTools)
-if(UNIX AND NOT APPLE)
-    find_package(Qt6 COMPONENTS ${QT_COMPONENTS} DBus REQUIRED)
-elseif(APPLE)
+if(APPLE)
     find_package(Qt6 COMPONENTS ${QT_COMPONENTS} REQUIRED HINTS
             /usr/local/opt/qt/lib/cmake
             /usr/local/Cellar/qt/*/lib/cmake
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 1bf2a79..62f1790 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -243,20 +243,6 @@ if(UNIX AND NOT APPLE)
         list(APPEND gui_SOURCES
             gui/osutils/nixutils/X11Funcs.cpp)
     endif()
-    qt6_add_dbus_adaptor(gui_SOURCES
-            gui/org.keepassxc.KeePassXC.MainWindow.xml
-            gui/MainWindow.h
-            MainWindow)
-
-    set_source_files_properties(
-        quickunlock/dbus/org.freedesktop.PolicyKit1.Authority.xml
-        PROPERTIES
-        INCLUDE "quickunlock/PolkitDbusTypes.h"
-        )
-    qt6_add_dbus_interface(core_SOURCES
-        quickunlock/dbus/org.freedesktop.PolicyKit1.Authority.xml
-        polkit_dbus
-    )
 endif()
 
 if(WIN32)
@@ -364,7 +350,7 @@ if(HAIKU)
     target_link_libraries(keepassxc_gui network)
 endif()
 if(UNIX AND NOT APPLE)
-    target_link_libraries(keepassxc_core Qt6::DBus ${LIBUSB_LIBRARIES})
+    target_link_libraries(keepassxc_core ${LIBUSB_LIBRARIES})
     if("${CMAKE_SYSTEM}" MATCHES "Linux")
         target_link_libraries(keepassxc_core ${KEYUTILS_LIBRARIES})
     endif()
diff --git a/src/gui/DatabaseOpenWidget.cpp b/src/gui/DatabaseOpenWidget.cpp
index 8d3d0b4..a51606a 100644
--- a/src/gui/DatabaseOpenWidget.cpp
+++ b/src/gui/DatabaseOpenWidget.cpp
@@ -39,9 +39,6 @@ namespace
 
     bool isQuickUnlockAvailable()
     {
-        if (config()->get(Config::Security_QuickUnlock).toBool()) {
-            return getQuickUnlock()->isAvailable();
-        }
         return false;
     }
 } // namespace
@@ -374,7 +371,7 @@ void DatabaseOpenWidget::openDatabase()
         // Save Quick Unlock credentials if available
         if (!blockQuickUnlock && isQuickUnlockAvailable()) {
             auto keyData = databaseKey->serialize();
-            getQuickUnlock()->setKey(m_db->publicUuid(), keyData);
+            //getQuickUnlock()->setKey(m_db->publicUuid(), keyData);
             m_ui->messageWidget->hideMessage();
         }
 
@@ -422,12 +419,12 @@ QSharedPointer<CompositeKey> DatabaseOpenWidget::buildDatabaseKey()
     if (!m_db.isNull() && canPerformQuickUnlock()) {
         // try to retrieve the stored password using Windows Hello
         QByteArray keyData;
-        if (!getQuickUnlock()->getKey(m_db->publicUuid(), keyData)) {
-            m_ui->messageWidget->showMessage(
-                tr("Failed to authenticate with Quick Unlock: %1").arg(getQuickUnlock()->errorString()),
-                MessageWidget::Error);
-            return {};
-        }
+        //if (!getQuickUnlock()->getKey(m_db->publicUuid(), keyData)) {
+        //    m_ui->messageWidget->showMessage(
+        //        tr("Failed to authenticate with Quick Unlock: %1").arg(getQuickUnlock()->errorString()),
+        //        MessageWidget::Error);
+        //    return {};
+        //}
         databaseKey->setRawKey(keyData);
         return databaseKey;
     }
@@ -612,7 +609,8 @@ void DatabaseOpenWidget::setUserInteractionLock(bool state)
 
 bool DatabaseOpenWidget::canPerformQuickUnlock() const
 {
-    return !m_db.isNull() && isQuickUnlockAvailable() && getQuickUnlock()->hasKey(m_db->publicUuid());
+    return false;
+    //return !m_db.isNull() && isQuickUnlockAvailable() && getQuickUnlock()->hasKey(m_db->publicUuid());
 }
 
 bool DatabaseOpenWidget::isOnQuickUnlockScreen() const
@@ -655,7 +653,7 @@ void DatabaseOpenWidget::resetQuickUnlock()
         return;
     }
     if (!m_db.isNull()) {
-        getQuickUnlock()->reset(m_db->publicUuid());
+        //getQuickUnlock()->reset(m_db->publicUuid());
     }
     load(m_filename);
 }
diff --git a/src/gui/MainWindow.cpp b/src/gui/MainWindow.cpp
index 9b39a62..7414b79 100644
--- a/src/gui/MainWindow.cpp
+++ b/src/gui/MainWindow.cpp
@@ -72,9 +72,9 @@
 #include "browser/BrowserService.h"
 #endif
 
-#if defined(Q_OS_UNIX) && !defined(Q_OS_MACOS) && !defined(QT_NO_DBUS)
-#include "mainwindowadaptor.h"
-#endif
+//#if defined(Q_OS_UNIX) && !defined(Q_OS_MACOS) && !defined(QT_NO_DBUS)
+//#include "mainwindowadaptor.h"
+//#endif
 
 const QString MainWindow::BaseWindowTitle = "KeePassXC";
 
@@ -95,12 +95,12 @@ MainWindow::MainWindow()
     macUtils()->configureWindowAndHelpMenus(this, m_ui->menuHelp);
 #endif
 
-#if defined(Q_OS_UNIX) && !defined(Q_OS_MACOS) && !defined(QT_NO_DBUS)
-    new MainWindowAdaptor(this);
-    QDBusConnection dbus = QDBusConnection::sessionBus();
-    dbus.registerObject("/keepassxc", this);
-    dbus.registerService("org.keepassxc.KeePassXC.MainWindow");
-#endif
+//#if defined(Q_OS_UNIX) && !defined(Q_OS_MACOS) && !defined(QT_NO_DBUS)
+//    new MainWindowAdaptor(this);
+//    QDBusConnection dbus = QDBusConnection::sessionBus();
+//    dbus.registerObject("/keepassxc", this);
+//    dbus.registerService("org.keepassxc.KeePassXC.MainWindow");
+//#endif
 
     setAcceptDrops(true);
 
diff --git a/src/gui/dbsettings/DatabaseSettingsWidgetDatabaseKey.cpp b/src/gui/dbsettings/DatabaseSettingsWidgetDatabaseKey.cpp
index b41ca1f..667ac7b 100644
--- a/src/gui/dbsettings/DatabaseSettingsWidgetDatabaseKey.cpp
+++ b/src/gui/dbsettings/DatabaseSettingsWidgetDatabaseKey.cpp
@@ -215,7 +215,7 @@ bool DatabaseSettingsWidgetDatabaseKey::saveSettings()
 
     m_db->setKey(newKey, true, false, false);
 
-    getQuickUnlock()->reset(m_db->publicUuid());
+    //getQuickUnlock()->reset(m_db->publicUuid());
 
     emit editFinished(true);
     if (m_isDirty) {
diff --git a/src/gui/osutils/ScreenLockListenerPrivate.cpp b/src/gui/osutils/ScreenLockListenerPrivate.cpp
index 76df0a9..3f5c011 100644
--- a/src/gui/osutils/ScreenLockListenerPrivate.cpp
+++ b/src/gui/osutils/ScreenLockListenerPrivate.cpp
@@ -19,9 +19,7 @@
 #if defined(Q_OS_MACOS)
 #include "macutils/ScreenLockListenerMac.h"
 #elif defined(Q_OS_UNIX)
-#include "nixutils/ScreenLockListenerDBus.h"
-
-#include <QWidget>
+//#include "nixutils/ScreenLockListenerDBus.h"
 #elif defined(Q_OS_WIN)
 #include "winutils/ScreenLockListenerWin.h"
 #endif
@@ -37,7 +35,8 @@ ScreenLockListenerPrivate* ScreenLockListenerPrivate::instance(QWidget* parent)
     Q_UNUSED(parent);
     return ScreenLockListenerMac::instance();
 #elif defined(Q_OS_UNIX)
-    return new ScreenLockListenerDBus(parent);
+    //return new ScreenLockListenerDBus(parent);
+    return nullptr;
 #elif defined(Q_OS_WIN)
     return new ScreenLockListenerWin(parent);
 #endif
diff --git a/src/gui/osutils/nixutils/NixUtils.cpp b/src/gui/osutils/nixutils/NixUtils.cpp
index dbdc7bd..24f54ac 100644
--- a/src/gui/osutils/nixutils/NixUtils.cpp
+++ b/src/gui/osutils/nixutils/NixUtils.cpp
@@ -22,7 +22,6 @@
 #include "core/Global.h"
 
 #include <QApplication>
-#include <QDBusInterface>
 #include <QDebug>
 #include <QDir>
 #include <QPointer>
@@ -73,18 +72,18 @@ NixUtils::NixUtils(QObject* parent)
 #endif
 
     // notify about system color scheme changes
-    QDBusConnection sessionBus = QDBusConnection::sessionBus();
-    sessionBus.connect("org.freedesktop.portal.Desktop",
-                       "/org/freedesktop/portal/desktop",
-                       "org.freedesktop.portal.Settings",
-                       "SettingChanged",
-                       this,
-                       SLOT(handleColorSchemeChanged(QString, QString, QDBusVariant)));
-
-    QDBusMessage msg = QDBusMessage::createMethodCall(
-        "org.freedesktop.portal.Desktop", "/org/freedesktop/portal/desktop", "org.freedesktop.portal.Settings", "Read");
-    msg << QVariant("org.freedesktop.appearance") << QVariant("color-scheme");
-    sessionBus.callWithCallback(msg, this, SLOT(handleColorSchemeRead(QDBusVariant)));
+    //QDBusConnection sessionBus = QDBusConnection::sessionBus();
+    //sessionBus.connect("org.freedesktop.portal.Desktop",
+    //                   "/org/freedesktop/portal/desktop",
+    //                   "org.freedesktop.portal.Settings",
+    //                   "SettingChanged",
+    //                   this,
+    //                   SLOT(handleColorSchemeChanged(QString, QString, QDBusVariant)));
+
+    //QDBusMessage msg = QDBusMessage::createMethodCall(
+    //    "org.freedesktop.portal.Desktop", "/org/freedesktop/portal/desktop", "org.freedesktop.portal.Settings", "Read");
+    //msg << QVariant("org.freedesktop.appearance") << QVariant("color-scheme");
+    //sessionBus.callWithCallback(msg, this, SLOT(handleColorSchemeRead(QDBusVariant)));
 }
 
 NixUtils::~NixUtils() = default;
@@ -170,6 +169,7 @@ void NixUtils::setLaunchAtStartup(bool enable)
         QFile::remove(getAutostartDesktopFilename());
     }
 #else
+    /*
     QDBusConnection sessionBus = QDBusConnection::sessionBus();
     QDBusMessage msg = QDBusMessage::createMethodCall("org.freedesktop.portal.Desktop",
                                                       "/org/freedesktop/portal/desktop",
@@ -198,17 +198,21 @@ void NixUtils::setLaunchAtStartup(bool enable)
     if (!res) {
         qDebug() << "DBus Error: could not connect to org.freedesktop.portal.Request";
     }
+    */
+    qDebug() << "This is not implemented as DBus is patched out.";
 #endif
 }
 
 void NixUtils::launchAtStartupRequested(uint response, const QVariantMap& results)
 {
-    if (response > 0) {
-        qDebug() << "DBus Error: the request to autostart was cancelled.";
-        return;
-    }
-
-    config()->set(Config::GUI_LaunchAtStartup, results["autostart"].value<bool>());
+    //if (response > 0) {
+    //    qDebug() << "DBus Error: the request to autostart was cancelled.";
+    //    return;
+    //}
+
+    //config()->set(Config::GUI_LaunchAtStartup, results["autostart"].value<bool>());
+    Q_UNUSED(response)
+    Q_UNUSED(results)
 }
 
 bool NixUtils::isCapslockEnabled()
@@ -359,24 +363,25 @@ bool NixUtils::unregisterGlobalShortcut(const QString& name)
     return true;
 }
 
-void NixUtils::handleColorSchemeRead(QDBusVariant value)
+void NixUtils::handleColorSchemeRead()
 {
-    value = qvariant_cast<QDBusVariant>(value.variant());
-    setColorScheme(value);
+    //value = qvariant_cast<QDBusVariant>(value.variant());
+    //setColorScheme(value);
 }
 
-void NixUtils::handleColorSchemeChanged(QString ns, QString key, QDBusVariant value)
+void NixUtils::handleColorSchemeChanged()
 {
-    if (ns == "org.freedesktop.appearance" && key == "color-scheme") {
-        setColorScheme(value);
-    }
+    //if (ns == "org.freedesktop.appearance" && key == "color-scheme") {
+    //    setColorScheme(value);
+    //}
 }
 
-void NixUtils::setColorScheme(QDBusVariant value)
+void NixUtils::setColorScheme(QString value)
 {
-    m_systemColorschemePref = static_cast<ColorschemePref>(value.variant().toInt());
-    m_systemColorschemePrefExists = true;
-    emit interfaceThemeChanged();
+    Q_UNUSED(value)
+    //m_systemColorschemePref = static_cast<ColorschemePref>(value.variant().toInt());
+    //m_systemColorschemePrefExists = true;
+    //emit interfaceThemeChanged();
 }
 
 quint64 NixUtils::getProcessStartTime() const
diff --git a/src/gui/osutils/nixutils/NixUtils.h b/src/gui/osutils/nixutils/NixUtils.h
index 98f1df7..b72c53c 100644
--- a/src/gui/osutils/nixutils/NixUtils.h
+++ b/src/gui/osutils/nixutils/NixUtils.h
@@ -20,8 +20,8 @@
 
 #include "gui/osutils/OSUtilsBase.h"
 #include <QAbstractNativeEventFilter>
+#include <QHash>
 #include <QSharedPointer>
-#include <QtDBus/QDBusVariant>
 
 class NixUtils : public OSUtilsBase, QAbstractNativeEventFilter
 {
@@ -53,8 +53,8 @@ public:
     quint64 getProcessStartTime() const;
 
 private slots:
-    void handleColorSchemeRead(QDBusVariant value);
-    void handleColorSchemeChanged(QString ns, QString key, QDBusVariant value);
+    void handleColorSchemeRead();
+    void handleColorSchemeChanged();
     void launchAtStartupRequested(uint response, const QVariantMap& results);
 
 private:
@@ -86,7 +86,7 @@ private:
     ColorschemePref m_systemColorschemePref = ColorschemePref::PreferNone;
     bool m_systemColorschemePrefExists = false;
 
-    void setColorScheme(QDBusVariant value);
+    void setColorScheme(QString value);
 
     Q_DISABLE_COPY(NixUtils)
 };
diff --git a/src/gui/osutils/nixutils/ScreenLockListenerDBus.cpp b/src/gui/osutils/nixutils/ScreenLockListenerDBus.cpp
index 1da87b9..a6db2eb 100644
--- a/src/gui/osutils/nixutils/ScreenLockListenerDBus.cpp
+++ b/src/gui/osutils/nixutils/ScreenLockListenerDBus.cpp
@@ -17,13 +17,13 @@
 
 #include "ScreenLockListenerDBus.h"
 
-#include <QDBusInterface>
 #include <QDebug>
 #include <QProcessEnvironment>
 
 ScreenLockListenerDBus::ScreenLockListenerDBus(QWidget* parent)
     : ScreenLockListenerPrivate(parent)
 {
+/*
     QDBusConnection sessionBus = QDBusConnection::sessionBus();
     QDBusConnection systemBus = QDBusConnection::systemBus();
 
@@ -78,10 +78,12 @@ ScreenLockListenerDBus::ScreenLockListenerDBus(QWidget* parent)
                        "Locked", // signal name
                        this, // receiver
                        SLOT(unityLocked()));
+*/
 }
 
-void ScreenLockListenerDBus::login1SessionObjectReceived(QDBusMessage response)
+void ScreenLockListenerDBus::login1SessionObjectReceived(/*QDBusMessage response*/)
 {
+/*
     if (response.arguments().isEmpty()) {
         qDebug() << "org.freedesktop.login1.Manager.GetSession did not return results";
         return;
@@ -100,30 +102,31 @@ void ScreenLockListenerDBus::login1SessionObjectReceived(QDBusMessage response)
                       "Lock", // signal name
                       this, // receiver
                       SLOT(unityLocked()));
+*/
 }
 
 void ScreenLockListenerDBus::gnomeSessionStatusChanged(uint status)
 {
-    if (status != 0) {
-        emit screenLocked();
-    }
+    //if (status != 0) {
+    //    emit screenLocked();
+    //}
 }
 
 void ScreenLockListenerDBus::logindPrepareForSleep(bool beforeSleep)
 {
-    if (beforeSleep) {
-        emit screenLocked();
-    }
+    //if (beforeSleep) {
+    //    emit screenLocked();
+    //}
 }
 
 void ScreenLockListenerDBus::unityLocked()
 {
-    emit screenLocked();
+    //emit screenLocked();
 }
 
 void ScreenLockListenerDBus::freedesktopScreenSaver(bool status)
 {
-    if (status) {
-        emit screenLocked();
-    }
+    //if (status) {
+    //   emit screenLocked();
+    //}
 }
diff --git a/src/gui/osutils/nixutils/ScreenLockListenerDBus.h b/src/gui/osutils/nixutils/ScreenLockListenerDBus.h
index 4ece813..5ef26b8 100644
--- a/src/gui/osutils/nixutils/ScreenLockListenerDBus.h
+++ b/src/gui/osutils/nixutils/ScreenLockListenerDBus.h
@@ -19,7 +19,6 @@
 #define SCREENLOCKLISTENERDBUS_H
 
 #include "gui/osutils/ScreenLockListenerPrivate.h"
-#include <QDBusMessage>
 
 class ScreenLockListenerDBus : public ScreenLockListenerPrivate
 {
@@ -32,7 +31,7 @@ private slots:
     void logindPrepareForSleep(bool beforeSleep);
     void unityLocked();
     void freedesktopScreenSaver(bool status);
-    void login1SessionObjectReceived(QDBusMessage);
+    void login1SessionObjectReceived(/*QDBusMessage*/);
 };
 
 #endif // SCREENLOCKLISTENERDBUS_H
diff --git a/src/quickunlock/Polkit.cpp b/src/quickunlock/Polkit.cpp
index d73a7c7..ba2cf6f 100644
--- a/src/quickunlock/Polkit.cpp
+++ b/src/quickunlock/Polkit.cpp
@@ -24,7 +24,6 @@
 
 #include <QDebug>
 #include <QFile>
-#include <QtDBus>
 #include <botan/mem_ops.h>
 #include <cerrno>
 
@@ -37,17 +36,17 @@ const QString polkit_object = "/org/freedesktop/PolicyKit1/Authority";
 
 namespace
 {
-    QString getKeyName(const QUuid& dbUuid)
-    {
-        static const QString keyPrefix = "keepassxc_polkit_keys_";
-        return keyPrefix + dbUuid.toString();
-    }
+    //QString getKeyName(const QUuid& dbUuid)
+    //{
+    //    static const QString keyPrefix = "keepassxc_polkit_keys_";
+    //    return keyPrefix + dbUuid.toString();
+    //}
 } // namespace
 
 Polkit::Polkit()
 {
-    PolkitSubject::registerMetaType();
-    PolkitAuthorizationResults::registerMetaType();
+    //PolkitSubject::registerMetaType();
+    //PolkitAuthorizationResults::registerMetaType();
 
     /* Note we explicitly use our own dbus path here, as the ::systemBus() method could be overridden
        through an environment variable to return an alternative bus path. This bus could have an application
@@ -56,37 +55,37 @@ Polkit::Polkit()
        Most Linux distros place the system bus at this exact path, so it is hard-coded.
        For any other distros, this path will need to be patched before compilation.
     */
-    QDBusConnection bus =
-        QDBusConnection::connectToBus("unix:path=/run/dbus/system_bus_socket", "keepassxc_polkit_dbus");
+    //QDBusConnection bus =
+    //    QDBusConnection::connectToBus("unix:path=/run/dbus/system_bus_socket", "keepassxc_polkit_dbus");
 
-    m_available = bus.isConnected();
-    if (!m_available) {
-        qDebug() << "polkit: Failed to connect to system dbus (this may be due to a non-standard dbus path)";
-        return;
-    }
+    //m_available = bus.isConnected();
+    //if (!m_available) {
+    //    qDebug() << "polkit: Failed to connect to system dbus (this may be due to a non-standard dbus path)";
+    return;
+    //}
 
-    m_available = bus.interface()->isServiceRegistered(polkit_service);
+    //m_available = bus.interface()->isServiceRegistered(polkit_service);
 
-    if (!m_available) {
-        qDebug() << "polkit: Polkit is not registered on dbus";
-        return;
-    }
+    //if (!m_available) {
+    //    qDebug() << "polkit: Polkit is not registered on dbus";
+    //    return;
+    //}
 
-    m_polkit.reset(new org::freedesktop::PolicyKit1::Authority(polkit_service, polkit_object, bus));
+    //m_polkit.reset(new org::freedesktop::PolicyKit1::Authority(polkit_service, polkit_object, bus));
 }
 
 Polkit::~Polkit()
 {
 }
 
-void Polkit::reset(const QUuid& dbUuid)
-{
-    m_encryptedMasterKeys.remove(dbUuid);
-}
+//void Polkit::reset(/*const QUuid& dbUuid*/)
+//{
+//    //m_encryptedMasterKeys.remove(dbUuid);
+//}
 
 bool Polkit::isAvailable() const
 {
-    return m_available;
+    return false;
 }
 
 QString Polkit::errorString() const
@@ -96,11 +95,12 @@ QString Polkit::errorString() const
 
 void Polkit::reset()
 {
-    m_encryptedMasterKeys.clear();
+    //m_encryptedMasterKeys.clear();
 }
 
-bool Polkit::setKey(const QUuid& dbUuid, const QByteArray& key)
+bool Polkit::setKey(/*const QUuid& dbUuid, const QByteArray& key*/)
 {
+    /*
     reset(dbUuid);
 
     // Generate a random iv/key pair to encrypt the master password with
@@ -142,10 +142,13 @@ bool Polkit::setKey(const QUuid& dbUuid, const QByteArray& key)
     // Store encrypted master password and return
     m_encryptedMasterKeys.insert(dbUuid, encryptedMasterKey);
     return true;
+    */
+    return false;
 }
 
-bool Polkit::getKey(const QUuid& dbUuid, QByteArray& key)
+bool Polkit::getKey(/*const QUuid& dbUuid, QByteArray& key*/)
 {
+    /*
     if (!m_polkit || !hasKey(dbUuid)) {
         return false;
     }
@@ -233,15 +236,15 @@ bool Polkit::getKey(const QUuid& dbUuid, QByteArray& key)
     } else {
         m_error = QObject::tr("Polkit authorization failed");
     }
-
+    */
     return false;
 }
 
-bool Polkit::hasKey(const QUuid& dbUuid) const
+bool Polkit::hasKey(/*const QUuid& dbUuid*/) const
 {
-    if (!m_encryptedMasterKeys.contains(dbUuid)) {
+    //if (!m_encryptedMasterKeys.contains(dbUuid)) {
         return false;
-    }
+    //}
 
-    return find_key_by_type_and_desc("user", getKeyName(dbUuid).toStdString().c_str(), KEY_SPEC_PROCESS_KEYRING) != -1;
+    //return find_key_by_type_and_desc("user", getKeyName(dbUuid).toStdString().c_str(), KEY_SPEC_PROCESS_KEYRING) != -1;
 }
diff --git a/src/quickunlock/Polkit.h b/src/quickunlock/Polkit.h
index 17bef55..297f2f2 100644
--- a/src/quickunlock/Polkit.h
+++ b/src/quickunlock/Polkit.h
@@ -19,7 +19,7 @@
 #define KEEPASSXC_POLKIT_H
 
 #include "QuickUnlockInterface.h"
-#include "polkit_dbus.h"
+//#include "polkit_dbus.h"
 #include <QHash>
 #include <QScopedPointer>
 
@@ -32,11 +32,10 @@ public:
     bool isAvailable() const override;
     QString errorString() const override;
 
-    bool setKey(const QUuid& dbUuid, const QByteArray& key) override;
-    bool getKey(const QUuid& dbUuid, QByteArray& key) override;
-    bool hasKey(const QUuid& dbUuid) const override;
+    bool setKey(/*const QUuid& dbUuid, const QByteArray& key*/);
+    bool getKey(/*const QUuid& dbUuid, QByteArray& key*/);
+    bool hasKey(/*const QUuid& dbUuid*/) const;
 
-    void reset(const QUuid& dbUuid) override;
     void reset() override;
 
 private:
@@ -44,7 +43,7 @@ private:
     QString m_error;
     QHash<QUuid, QByteArray> m_encryptedMasterKeys;
 
-    QScopedPointer<org::freedesktop::PolicyKit1::Authority> m_polkit;
+    //QScopedPointer<org::freedesktop::PolicyKit1::Authority> m_polkit;
 };
 
 #endif // KEEPASSXC_POLKIT_H
diff --git a/src/quickunlock/PolkitDbusTypes.cpp b/src/quickunlock/PolkitDbusTypes.cpp
index a4305dc..545463b 100644
--- a/src/quickunlock/PolkitDbusTypes.cpp
+++ b/src/quickunlock/PolkitDbusTypes.cpp
@@ -2,10 +2,11 @@
 
 void PolkitSubject::registerMetaType()
 {
-    qRegisterMetaType<PolkitSubject>("PolkitSubject");
-    qDBusRegisterMetaType<PolkitSubject>();
+    //qRegisterMetaType<PolkitSubject>("PolkitSubject");
+    //qDBusRegisterMetaType<PolkitSubject>();
 }
 
+/*
 QDBusArgument& operator<<(QDBusArgument& argument, const PolkitSubject& subject)
 {
     argument.beginStructure();
@@ -21,13 +22,14 @@ const QDBusArgument& operator>>(const QDBusArgument& argument, PolkitSubject& su
     argument.endStructure();
     return argument;
 }
-
+*/
 void PolkitAuthorizationResults::registerMetaType()
 {
-    qRegisterMetaType<PolkitAuthorizationResults>("PolkitAuthorizationResults");
-    qDBusRegisterMetaType<PolkitAuthorizationResults>();
+    //qRegisterMetaType<PolkitAuthorizationResults>("PolkitAuthorizationResults");
+    //qDBusRegisterMetaType<PolkitAuthorizationResults>();
 }
 
+/*
 QDBusArgument& operator<<(QDBusArgument& argument, const PolkitAuthorizationResults& res)
 {
     argument.beginStructure();
@@ -43,3 +45,4 @@ const QDBusArgument& operator>>(const QDBusArgument& argument, PolkitAuthorizati
     argument.endStructure();
     return argument;
 }
+*/
diff --git a/src/quickunlock/PolkitDbusTypes.h b/src/quickunlock/PolkitDbusTypes.h
index 8be977a..7e88413 100644
--- a/src/quickunlock/PolkitDbusTypes.h
+++ b/src/quickunlock/PolkitDbusTypes.h
@@ -18,19 +18,17 @@
 #ifndef KEEPASSXC_POLKITDBUSTYPES_H
 #define KEEPASSXC_POLKITDBUSTYPES_H
 
-#include <QtDBus>
-
 class PolkitSubject
 {
 public:
-    QString kind;
-    QVariantMap details;
+    //QString kind;
+    //QVariantMap details;
 
     static void registerMetaType();
 
-    friend QDBusArgument& operator<<(QDBusArgument& argument, const PolkitSubject& subject);
+    //friend QDBusArgument& operator<<(QDBusArgument& argument, const PolkitSubject& subject);
 
-    friend const QDBusArgument& operator>>(const QDBusArgument& argument, PolkitSubject& subject);
+    //friend const QDBusArgument& operator>>(const QDBusArgument& argument, PolkitSubject& subject);
 };
 
 class PolkitAuthorizationResults
@@ -38,16 +36,16 @@ class PolkitAuthorizationResults
 public:
     bool is_authorized;
     bool is_challenge;
-    QMap<QString, QString> details;
+    //QMap<QString, QString> details;
 
     static void registerMetaType();
 
-    friend QDBusArgument& operator<<(QDBusArgument& argument, const PolkitAuthorizationResults& subject);
+    //friend QDBusArgument& operator<<(QDBusArgument& argument, const PolkitAuthorizationResults& subject);
 
-    friend const QDBusArgument& operator>>(const QDBusArgument& argument, PolkitAuthorizationResults& subject);
+    //friend const QDBusArgument& operator>>(const QDBusArgument& argument, PolkitAuthorizationResults& subject);
 };
 
-Q_DECLARE_METATYPE(PolkitSubject);
-Q_DECLARE_METATYPE(PolkitAuthorizationResults);
+//Q_DECLARE_METATYPE(PolkitSubject);
+//Q_DECLARE_METATYPE(PolkitAuthorizationResults);
 
 #endif // KEEPASSXC_POLKITDBUSTYPES_H
diff --git a/src/quickunlock/QuickUnlockInterface.cpp b/src/quickunlock/QuickUnlockInterface.cpp
index 0e24736..b2a37b7 100644
--- a/src/quickunlock/QuickUnlockInterface.cpp
+++ b/src/quickunlock/QuickUnlockInterface.cpp
@@ -51,31 +51,31 @@ QString NoQuickUnlock::errorString() const
     return QObject::tr("No Quick Unlock provider is available");
 }
 
-void NoQuickUnlock::reset()
-{
-}
+//void NoQuickUnlock::reset()
+//{
+//}
 
-bool NoQuickUnlock::setKey(const QUuid& dbUuid, const QByteArray& key)
+bool NoQuickUnlock::setKey(/*const QUuid& dbUuid, const QByteArray& key*/)
 {
-    Q_UNUSED(dbUuid)
-    Q_UNUSED(key)
+    //Q_UNUSED(dbUuid)
+    //Q_UNUSED(key)
     return false;
 }
 
-bool NoQuickUnlock::getKey(const QUuid& dbUuid, QByteArray& key)
+bool NoQuickUnlock::getKey(/*const QUuid& dbUuid, QByteArray& key*/)
 {
-    Q_UNUSED(dbUuid)
-    Q_UNUSED(key)
+    //Q_UNUSED(dbUuid)
+    //Q_UNUSED(key)
     return false;
 }
 
-bool NoQuickUnlock::hasKey(const QUuid& dbUuid) const
+bool NoQuickUnlock::hasKey(/*const QUuid& dbUuid*/) const
 {
-    Q_UNUSED(dbUuid)
+    //Q_UNUSED(dbUuid)
     return false;
 }
 
-void NoQuickUnlock::reset(const QUuid& dbUuid)
+void NoQuickUnlock::reset(/*const QUuid& dbUuid*/)
 {
-    Q_UNUSED(dbUuid)
+    //Q_UNUSED(dbUuid)
 }
diff --git a/src/quickunlock/QuickUnlockInterface.h b/src/quickunlock/QuickUnlockInterface.h
index 54aeb8a..3833be2 100644
--- a/src/quickunlock/QuickUnlockInterface.h
+++ b/src/quickunlock/QuickUnlockInterface.h
@@ -31,12 +31,12 @@ public:
     virtual bool isAvailable() const = 0;
     virtual QString errorString() const = 0;
 
-    virtual bool setKey(const QUuid& dbUuid, const QByteArray& key) = 0;
-    virtual bool getKey(const QUuid& dbUuid, QByteArray& key) = 0;
-    virtual bool hasKey(const QUuid& dbUuid) const = 0;
+    virtual bool setKey(/*const QUuid& dbUuid, const QByteArray& key*/) = 0;
+    virtual bool getKey(/*const QUuid& dbUuid, QByteArray& key*/) = 0;
+    virtual bool hasKey(/*const QUuid& dbUuid*/) const = 0;
 
-    virtual void reset(const QUuid& dbUuid) = 0;
-    virtual void reset() = 0;
+    virtual void reset(/*const QUuid& dbUuid*/) = 0;
+    //virtual void reset() = 0;
 };
 
 class NoQuickUnlock : public QuickUnlockInterface
@@ -45,11 +45,11 @@ public:
     bool isAvailable() const override;
     QString errorString() const override;
 
-    bool setKey(const QUuid& dbUuid, const QByteArray& key) override;
-    bool getKey(const QUuid& dbUuid, QByteArray& key) override;
-    bool hasKey(const QUuid& dbUuid) const override;
+    bool setKey(/*const QUuid& dbUuid, const QByteArray& key*/);
+    bool getKey(/*const QUuid& dbUuid, QByteArray& key*/);
+    bool hasKey(/*const QUuid& dbUuid*/) const;
 
-    void reset(const QUuid& dbUuid) override;
+    //void reset(const QUuid& dbUuid) override;
     void reset() override;
 };
 
